crates/move-prover-boogie-backend/src/boogie_backend/bytecode_translator.rs

```diff
@@ -320,6 +323,15 @@ impl<'env> BoogieTranslator<'env> {
                 }
 
                 if self.targets.is_spec(&fun_env.get_qualified_id()) {
+                    if self.verify_only {
+                        self.translate_function_style(fun_env, FunctionTranslationStyle::Opaque);
+                        self.translate_function_style(fun_env, FunctionTranslationStyle::Aborts);
+                        if !self.targets.is_verified_spec(&fun_env.get_qualified_id()) {
+                            self.translate_function_style(fun_env, FunctionTranslationStyle::SpecNoAbortCheck);
+                        }
+                        continue;
+                    }
```

Comment:
let's move this with the changes below, to group the code for the function specs.